### PR TITLE
Bring back unit tests & linters to CI

### PR DESCRIPTION
**Pull request by zupo**
They were accidentally removed in e074e44#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47L70-L75

Refs https://github.com/Pylons/pyramid_openapi3/pull/132#issue-645943664

Also, update dependencies to latest stable versions.

### CODE REVIEW

**Review by myrheimb — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def run(self):  # noqa: D102
     install_requires=[
         "openapi-core>=0.13.4,<0.14",
         "openapi-spec-validator",
-        "pyramid",
+        "pyramid<2.0",
```
Comment (stevepiercy) [inline setup.py:75]:
```suggestion
        "pyramid>=1.10.7,<2.0",
```

**Review by stevepiercy — COMMENTED** [review]
To align with minimum Pyramid version

**Comment by lkuchenb** [comment]
Pyramid 2.0 is the latest stable version, yet now Pyramid < 2.0 is required. I've been using Pyramid 2.0 with 0.11, is there a known incompatibility?

**Comment by zupo** [comment]
Hey @lkuchenb, at the time I created this PR, at least some tests were failing when running against Pyramid 2.0, that is why we pinned it to < 2.0. Maybe the failures are no longer there, could you prepare a PR that bumps the support to 2.0?

**Comment by lkuchenb** [comment]
Sure, can do.

**Pull request merged**